src/Business Foundation/App/AuditCodes/src/SourceCodes.Page.al

```diff
@@ -0,0 +1,59 @@
+﻿// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.AuditCodes;
+
+/// <summary>
+/// Source codes are used to categorize the source of a transaction.
+/// </summary>
+page 257 "Source Codes"
+{
+    ApplicationArea = All;
+    Caption = 'Source Codes';
+    PageType = List;
+    SourceTable = "Source Code";
+    UsageCategory = Administration;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(Control1)
+            {
+                ShowCaption = false;
+                /// <summary>
+                /// Specifies the source code. 
+                /// </summary>
+                field("Code"; Rec.Code)
+                {
+                    ApplicationArea = All;
```

Comment:
@grobyns 
- When ApplicationArea is set on page level, is it still required to be set on page fields?
- For table fields with non-blank property, we mostly set the ShowMandatory property on page fields as well to highlight the required input?
- Will MS start using tooltips on tables (for new dev) or perform a full move if all tools are ready to do so?